itou/www/signup/views.py

```diff
@@ -624,6 +562,8 @@ def dispatch(self, request, *args, **kwargs):
 
         self.authorization_status = authorization_status
         self.kind = kind
+        # Get kind label
+        self.kind_label = dict(PrescriberOrganization.Kind.choices).get(kind)
```

Comment:
```suggestion
        self.kind_label = PrescriberOrganization.Kind[kind].label
```

Alternative pour le fun :)